lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -256,6 +260,12 @@ class AzurePlatformSchema:
     vm_tags: Optional[Dict[str, Any]] = field(default=None)
     locations: Optional[Union[str, List[str]]] = field(default=None)
 
+    virtual_network_resource_group: str = field(default="")
+    virtual_network_name: str = field(default=AZURE_NEW_VIRTUAL_NETWORK)
+    subnet_prefix: str = field(default=AZURE_NEW_SUBNET_PREFIX)
+    use_existing_virtual_network: bool = field(default=False)
+    use_private_address_for_vm_connection: bool = field(default=False)
```

Comment:
The name is too long, how about `use_private_ip`

Reply:
updated in https://github.com/microsoft/lisa/pull/2682